src/transformers/onnx/convert.py

```diff
@@ -110,6 +112,19 @@ def export(
         if not inputs_match:
             raise ValueError("Model and config inputs doesn't match")
 
+        if isinstance(model.base_model, LayoutLMv2Model):
```

Comment:
This is problematic because export should be somehow agnostic to the model to export.
If I understand correctly you want to replace `nn.AdaptiveAvgPool2d` by `nn.AvgPool2d`.
What you can do instead of patching the modules (also is it wanted to have hardcoded values for kernel size and stride?), is to patch the underlying op: `torch.nn.functional.adaptive_avg_pool2d` -> `torch.nn.functional.avg_pool2d`.

You can do this by providing a `PatchingSpec` in the `LayoutLMv2OnnxConfig.__init__` method.

Reply:
I will look into it.